ui/app/AppLayouts/Communities/popups/TransferOwnershipAlertPopup.qml

```diff
@@ -0,0 +1,72 @@
+import QtQuick 2.15
```

Comment:
`TransferOwnershipAlertPopup`, as the name suggests, is a very specific component for very specific need. So I would opt for encapsulating two variants of `headerTitle` and `alertText` in this component and expose only enum allowing to switch "mode" for that component. It will make that component more self-contained and also easier to test e.g. via storybook page (btw it would be nice to add sb page ;). That approach will also reduce code in OverviewSettingsPanel. Wdyt?

Reply:
Everything done!

https://github.com/status-im/status-desktop/assets/97019400/1a7dff53-2fa0-443a-a68c-e0105a9b7bc3